tokens/ease.yml

```diff
@@ -0,0 +1,10 @@
+props:
+  - name: ease-default
+    value: [0.4, 0.22, 0.28, 1]
```

Comment:
It'd be nice to have usage guidelines for each one (in a `comment` field). Note that we'll be enforcing such documentation in the future.

Reply:
Will this documentation follow some sort of structure? Similar to how readme files in Polaris are used to generate the style guide?

Reply:
We'll probably have something like Markdown to support complex usage guidelines, like in https://polaris.shopify.com/design/colors#section-color-usage – for now I think one-liners that go straight to the point would be great, like we do for icons in polaris-icons.shopify.com